tests/Unit/inc/functions/rocketGetDirectoryPHPFilesArray.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Unit tests for _rocket_get_directory_php_files_array().
+ *
+ * @package WP_Rocket\Tests\Unit\inc\functions
+ * @author  Caspar Green
+ * @since   ver 3.6.1
+ */
+
+namespace WP_Rocket\Tests\Unit\inc\functions;
+
+use WP_Rocket\Tests\Unit\FilesystemTestCase;
+
+/**
+ * Test _rocket_get_directory_php_files_array().
+ *
+ * @package WP_Rocket\Tests\Unit\inc\functions
+ * @covers ::_rocket_get_config_path_php_files()
+ * @author  Caspar Green
+ * @since   ver 3.6.1
+ */
```

Comment:
Our standard for documenting test classes is this:

- `@covers`
- (Optional) `@uses`
- blank line
- `@group`

[Check out this example](https://github.com/wp-media/wp-rocket/blob/develop/tests/Unit/inc/functions/rocketCleanFiles.php#L10-L24).

Reply:
Done.